recipe/meta.yaml

```diff
@@ -11,44 +11,71 @@ source:
   sha256: 57892c345fb9c88e2ca54fda1b7bca3909c8936393c6aab23d0410fc4d149345
 
 build:
-  number: 4
-
+  number: 5
   script:
     - {{ PYTHON }} -m pip install . -vv  # [not win]
     - set PSP_GENERATOR=Visual Studio 14 2015  # [win]
     - {{ PYTHON }} setup.py build_ext install --single-version-externally-managed  --record=record.txt  # [win]
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - cmake
-    - make  # [not win]
   host:
-    - boost
-    - nodejs
-    - numpy
-    - pip
-    - pybind11
     - python
-    - re2  # [not win]
-    - yarn
   run:
-    - ipywidgets >=7.5.1
-    - {{ pin_compatible('numpy') }}
-    - pandas >=0.22.0
     - python
-    - python-dateutil >=2.8.0
-    - tornado >=4.5.3
-    - traitlets >=4.3.2
-test:
-  imports:
-    - perspective
-    - perspective.table
+
+outputs:
+  - name: perspective
+
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - numpy                                  # [build_platform != target_platform]
+        - pybind11                               # [build_platform != target_platform]
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - make  # [not win]
+      host:
+        - boost
+        - nodejs
+        - numpy
+        - pip
+        - pybind11
+        - python
+        - re2  # [not win]
+        - yarn
+      run:
+        - ipywidgets >=7.5.1
+        - {{ pin_compatible('numpy') }}
+        - pandas >=0.22.0
+        - python
+        - python-dateutil >=2.8.0
+        - tornado >=4.5.3
+        - traitlets >=4.3.2
+      run_constrained:
+        - {{ pin_subpackage("perspective-python", max_pin="x.x.x") }}
+
+    test:
+      imports:
+        - perspective
+        - perspective.table
+
+  - name: perspective-python
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - python
```

Comment:
Not sure that it's necessary for the alias to have a direct dependency on python.

Reply:
strangely, this was necessary to use the `{{ PYTHON }}` jinja template in the other parts of the feedstock

Reply:
I also find that unexpected!